test/test_jsonReportSchema.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import os
+import datetime
+import json
+import jsonschema
+import pytest
+
+
+def test_jsonschema_version():
+    assert "3.2.0" == jsonschema.__version__
+
+
+def test_location_monthlyReport_schema():
+    assert os.path.exists("schemas/monthlyReport.json")
+
+
+def test_monthlyReport_is_valid_json():
```

Comment:
```suggestion
def test_monthlyReport_schema_is_valid_json():
```
Here you check that the schema is a valid json, not that a montly report file follows the monthly report schema, maybe this name would be better